packages/cf-component-box/src/Box.js

```diff
@@ -0,0 +1,117 @@
+import React, { PropTypes } from 'react';
+import { createComponent } from 'cf-style-container';
+
+const styles = (props) => ({
+  margin: props.margin || props.theme['margin'],
+  marginTop: props.marginTop || props.theme['marginTop'],
+  marginBottom: props.marginBottom || props.theme['marginBottom'],
+  marginLeft: props.marginLeft || props.theme['marginLeft'],
+  marginRight: props.marginRight || props.theme['marginRight'],
+
+  padding: props.padding || props.theme['padding'],
```

Comment:
Good point. I added checking for this via the 'conflictsWith' property in the propertySpec. It should throw an error now if you do this.